connector/connect/server/src/main/scala/org/apache/spark/sql/connect/ui/SparkConnectServerPage.scala

```diff
@@ -330,24 +328,10 @@ private[ui] class SqlStatsPagedTable(
       <td>
         {sqlStatsTableRow.sparkSessionTags.mkString(", ")}
       </td>
-      {errorMessageCell(Option(info.detail))}
+      {UIUtils.errorMessageCell(Option(info.detail).getOrElse(""))}
```

Comment:
Can `info.detail` be `null`? If it cannot, can we just pass `info.detail` to `errorMessageCell`?

Reply:
The variable is initially set to an empty string and is then assigned to the value of `e.errorMessage` when an error occurs. It appears that `e.errorMessage` can be null in certain situations.

Reply:
The variable is initially set to an empty string and is then assigned to the value of `e.errorMessage` when an error occurs. It appears that `e.errorMessage` can be null in certain situations.

Reply:
Hmm, one option could be make `errorMessageCell` null-safe, but I think it's O.K to leave it as it is.